Source/ablastr/fields/PoissonSolver.H

```diff
@@ -232,16 +234,22 @@ computePhi (
                                        "Embedded boundary solve requested but not compiled in");
 #endif
 
-#if !defined(ABLASTR_USE_FFT)
+#if (!defined(ABLASTR_USE_FFT)) && (!defined(ABLASTR_USE_HEFFTE))
```

Comment:
Same comment as above: since `is_igf_3d` is a run-time option, `ABLASTR_USE_FFT` is always required.